Dockerfile-msphpsql

```diff
@@ -56,7 +56,7 @@ WORKDIR $PHPSQLDIR/source/
 RUN chmod +x ./packagize.sh
 RUN /bin/bash -c "./packagize.sh"
 
-RUN echo "extension = pdo_sqlsrv.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+RUN echo "extension = pdo_sqlsrv.so" >> /etc/php/7.0/cli/conf.d/20-pdo_sqlsrv.ini
 RUN echo "extension = sqlsrv.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
 
```

Comment:
How does this affect PECL install and/or installation instructions?

Reply:
We should update the instructions to create a new pdo_sqlsrv.ini file...

Reply:
yes, that too, would you update README in this PR then?
also, what about PECL install?
just so you know, eventually, the installation instructions will be on MSDN

Reply:
David, so this also means changing our current test scripts in the test lab?

Reply:
Working on it currently